src/components/SearchableSelect.tsx

```
@@ -91,58 +92,93 @@ const SearchableSelect = <Option, IsMulti extends boolean = false, Group extends
         components={{
           IndicatorSeparator,
           DropdownIndicator,
+          ...(props?.components ? props.components : {}),
         }}
         classNames={{
-          control: ({ isFocused }) => {
-            if (hasError && isFocused) {
-              return '!border-2 !border-danger-500';
+          ...(props?.classNames ? props.classNames : {}),
+          control: (currentProps) => {
+            const userControl = props.classNames?.control as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)